python_scripts/AddressPoint.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+#
+#Short script for formating AddressPoint datafile
+#run using python AddressPoint.py -i <filnamn>
+#
+import codecs, sys, getopt
+
+def main(argv):
+    inputfile = ''
+    outputfile = ''
+    try:
+        opts, args = getopt.getopt(argv,"hi:")
+    except getopt.GetoptError:
+        print u'python AddressPoint.py -i <filnamn>'
+        sys.exit(2)
+    for opt, arg in opts:
+        if opt == '-h':
+            print u'python AddressPoint.py -i <filnamn>'
+            sys.exit()
+        elif opt == "-i":
+            inputfile = arg
+    if len(inputfile) > 0:
+        outputfile = u'%s-mod.txt' %inputfile[:-4]
```

Comment:
The last expression removes file extension, right? Use [os.path.splitext](https://docs.python.org/3.5/library/os.path.html#os.path.splitext) for robustness and readability.